po/hi.po

```diff
@@ -2726,7 +2690,7 @@ msgstr "जापान"
 #: js/utils/musicutils.js:1074
 #.TRANS: https://en.wikipedia.org/wiki/In_scale and https://en.wikipedia.org/wiki/Sakura_Sakura
 msgid "in"
-msgstr ""
+msgstr "में"
```

Comment:
This is not the preposition, "in". It is the name of a mode, as per the TRANS note.

Reply:
Will you be able to fix this?